From 80c028452000465c607cce054211075d44634996 Mon Sep 17 00:00:00 2001
From: moagrawa <moagrawa@redhat.com>
Date: Mon, 15 Jan 2018 18:21:27 +0530
Subject: [PATCH 126/128] spec: unpackaged files found for RHEL-7 client build

Problem: unpackages files found for RHEL-7 client build

Solution: Update glusterfs.specs.in to exclude unpackage files
Label: DOWNSTREAM ONLY

BUG: 1534530
Change-Id: I211a739fb5ce18011bc75fb36c6112cc802fccc9
Signed-off-by: moagrawa <moagrawa@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/127681
Tested-by: RHGS Build Bot <nigelb@redhat.com>
Reviewed-by: Atin Mukherjee <amukherj@redhat.com>
---
 glusterfs.spec.in | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/glusterfs.spec.in b/glusterfs.spec.in
index 2ac9a60b0..cb273f5 100644
--- a/glusterfs.spec.in
+++ b/glusterfs.spec.in
@@ -1219,6 +1219,8 @@ exit 0
 %exclude %{_libdir}/pkgconfig/libgfdb.pc
 %exclude %{_sbindir}/gluster-setgfid2path
 %exclude %{_mandir}/man8/gluster-setgfid2path.8*
+%exclude %{_datadir}/glusterfs/scripts/control-cpu-load.sh
+%exclude %{_datadir}/glusterfs/scripts/control-mem.sh
 %endif
 
 %files api
@@ -2179,6 +2181,9 @@ fi
 %endif
 
 %changelog
+* Tue Jan 16 2018 Mohit Agrawal <moagrawa@redhat.com>
+- Exclude control-cpu-load.sh and control-mem.sh from client builds
+
 * Wed Dec 20 2017 Milind Changire <mchangir@redhat.com>
 - Remove ExclusiveArch directive to help building on non-x86_64 arches (#1527772)
 
-- 
1.8.3.1

